New Unity Project/Assets/Script/Phone/Clock.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Globalization;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+
+public class Clock : MonoBehaviour
+{
+    private TextMeshProUGUI clockText;
+    
+    void Start()
+    {
+        clockText = GetComponent<TextMeshProUGUI>();
+    }
+    void Update()
+    {
+        clockText.text = DateTime.Now.ToString("hh:mm tt");
```

Comment:
Anders de format weghalen om iedereen bij te houden? Als het goed is pakt het dan je systeem voorkeur https://docs.microsoft.com/en-us/dotnet/api/system.datetime.tostring?view=netstandard-2.0